From 3deb9d0d4c51f82ffdbbbe4270a251233aeb8779 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 22 May 2022 03:25:39 +0200
Subject: [PATCH 320/515] arm64: dts: rk3399-pinephone-pro: Add i2c supply to
 wcam

Sensor depends on I2C bus to be powered up to work.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 51bea6fbfdd1..d872a8422983 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -896,6 +896,7 @@ wcam: camera@1a {
 		clock-names = "xvclk";
 
 		vif-supply = <&vcc1v8_dvp>;
+		i2c-supply = <&vcca1v8_codec>;
 		/*XXX: also depends on vcca1v8_codec for I2C bus power (currently always on) */
 
 		reset-gpios = <&gpio1 RK_PA0 GPIO_ACTIVE_LOW>;
-- 
2.35.3

